app/components/hiring_staff/inline_components/vacancies_count_heading_component.rb

```diff
@@ -0,0 +1,12 @@
+class HiringStaff::InlineComponents::VacanciesCountHeadingComponent < ViewComponent::Base
+  def initialize(vacancies: vacancies, selected_type: selected_type)
+    @vacancies = vacancies
+    @selected_type = selected_type
+  end
+
+  def call
+    t("jobs.#{@selected_type}_jobs_with_count",
```

Comment:
I think my earlier comment is still applicable, but it's slightly less pertinent now we're out of the template